File: movie-rating/MovieRating.Data/Entities/MovieStar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MovieRating.Data.Entities
{
    /* Because we don't use any authentification, this class will be used to store 
       User rating for movies 
    */
    public class MovieStar
    {
        public int Id { get; set; }
        public int MovieId { get; set; }
        public Movie Movie { get; set; }
        public int Stars { get; set; }
    }
}

File: movie-rating/MovieRating.Web/ClientApp/src/app/home/models/movie-star.ts
export class MovieStar {
  id: number;
  movieId: number;
  stars: number;
  constructor(id:number, stars: number, yourRateId:number) {
      this.id = yourRateId ?? 0;
      this.movieId = id;
      this.stars = stars;
  }
}
